From e57df4822112647178d0ee5aa48cf12e5a615ed6 Mon Sep 17 00:00:00 2001
From: Zheng Zucheng <zhengzucheng@huawei.com>
Date: Thu, 14 Oct 2021 16:57:37 +0800
Subject: [PATCH] Revert "[Huawei] sched: export sched_setscheduler symbol"
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: e57df4822112647178d0ee5aa48cf12e5a615ed6


Offering: HULK
hulk inclusion
category: feature
bugzilla: 55087 https://gitee.com/openeuler/kernel/issues/I4DDEL

--------------------------------

This reverts commit 5d7eabd898017e2bc2a767f085f314a3d7ce6542.
SEG determines that the sched_setscheduler interface is exported
by downstream users, hulk follows the community solution.

Signed-off-by: Zheng Zucheng <zhengzucheng@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Chen Jun <chenjun102@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 kernel/sched/core.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/sched/core.c b/kernel/sched/core.c
index 35d3240e7b85..cb5c2f9be849 100644
--- a/kernel/sched/core.c
+++ b/kernel/sched/core.c
@@ -5708,7 +5708,6 @@ int sched_setscheduler(struct task_struct *p, int policy,
 {
 	return _sched_setscheduler(p, policy, param, true);
 }
-EXPORT_SYMBOL_GPL(sched_setscheduler);
 
 int sched_setattr(struct task_struct *p, const struct sched_attr *attr)
 {
-- 
2.26.2

